dali/operators/util/get_property.cc

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/util/get_property.h"
+#include "dali/pipeline/data/types.h"
+
+namespace dali {
+
+DALI_SCHEMA(GetProperty)
+    .DocStr(
+        R"code(Returns a property of the tensor passed as an input.
```

Comment:
```suggestion
        R"code(Returns a property of the tensor passed as an input.

```
^^ The first line should be one simple sentence (the only visible in the operator table)

Reply:
Done